posthog/api/person.py

```diff
@@ -228,7 +228,7 @@ def get_funnel_actor_class(filter: Filter) -> Callable:
 
 class PersonViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     """
-    To create or update persons, use a PostHog library of your choice and [use an identify call](/product-analytics/identify). This API endpoint is only for reading and deleting.
+    This endpoint is meant for reading and deleting persons. To create or update persons, we recommend using the [capture API](https://posthog.com/docs/api/capture), the `$set` and `$unset` properties, or one of our SDKs.
```

Comment:
I have a plan, where we only allow person updates via `$set` or `$identify` and friends events, can we instead link people to the persons page https://posthog.com/docs/data/persons

Reply:
The people doc doesn't really talk about `$set` or `$identify`, maybe this one: https://posthog.com/docs/product-analytics/user-properties?

Reply:
yes, that looks like the one, can we make data/persons link to it?